db/re/item_db.txt

```diff
@@ -3509,7 +3509,7 @@
 5393,Valentine_Hat,Love Valentine's Hat,4,20,,200,,4,,0,0xFFFFFFFF,63,2,256,,0,0,397,{ bonus bMaxSPrate,7; bonus bMaxHPrate,7; },{},{}
 5394,Bubblegum_Lower,Bubblegum,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1,,0,0,0,{ bonus2 bExpAddClass,Class_All,100; },{},{}
 5395,Tiraya_Bonnet,Striped Hat,4,20,,500,,6,,1,0xFFFFFFFF,63,2,256,,50,1,398,{ bonus bMaxHP,100+(getrefine()*20); bonus bLuk,3; },{},{}
-5396,Jasper_Crest,Jasper Crest,4,20,,700,,6,,1,0xFFFFFFFF,63,2,256,,0,1,399,{ bonus bDex,3; },{},{}
+5396,Jasper_Crest,Jasper Crest,4,20,,700,,6,,1,0xFFFFFFFF,63,2,256,,0,1,399,{ bonus bDex,1; bonus2 bAddRace,RC_Undead,5; if (getrefine()<6) { bonus2 bAddEff2,Eff_Blind,10; } if (getrefine()>8) { bonus bDex,2; } },{},{}
```

Comment:
Do you mind adding an "else" for the second if, since it can never be < 6 and > 8.